localization/languages/no.json

```diff
@@ -0,0 +1,156 @@
+{
+    "name": "Norwegian Bokmål",
+    "identifier": "no",
```

Comment:
I'm somewhat confused about what the right identifier for this is. The Electron docs list "no" as a generic "norwegian" option, while "nb" corresponds to "norwegian (Bokmal)". On macOS, there are two separate "norwegian" and "norwegian bakmal" options listed in the preferences, but both seem to correspond to "nb". Given that Wikipedia says Bokmal is the most popular form, maybe we want to update buildLocalization.js to accept an array of identifiers, so that this file will be used for both?

Reply:
So, I know I'm comming at this from nowhere, but as a norwegian myself I would like to point out that the "correct" thing to do would be to **not** use the identifier "no" (norwegian) at all -- and then use "nb" for norwegian bokmål. If you at some point added norwegian nynorsk, the identifier should then be "nn". The reason for this is that -- though "bokmål" is more popular -- the two languages are *equal* and should thus not be treated as anything other than equals.

I know this seems like beeing way to strickt but, trust me, there is **a lot** of feelings involved in the whole "bokmål" vs. "nynorsk" discusion, and some people (myself very included) get extramely ofended when "bokmål" is refered to simply as "norwegian" in contexts like these.